.buildkite/pipeline.release.yml

```diff
@@ -0,0 +1,23 @@
+agents:
+  queue: hosted
+
+steps:
+  # This is most to prevent we accidentally tag things.
+  - block: "OK to release?"
+
+  - command: ".buildkite/steps/release-pypi"
+    label: ":pypi:"
+    if: build.tag != null
+    env:
+      # It should be a noop
+      TWINE_USERNAME: __token__
+    plugins:
+      - cluster-secrets#v1.0.0:
+          variables:
+            TWINE_PASSWORD: test-collector-python-pypi-api-token
```

Comment:
This pipeline isn't working yet as I am still pending on some plumbing works to get this secret provisioned. 

But merging this PR has no harm.

Reply:
I will use a follow-up PR to replace this with AWS SSM